ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -150,6 +150,9 @@ export default class SendAssetRow extends Component {
       if (token) {
         return this.renderToken(token);
       }
+
+      // if token is not found in tokens list, render the details
+      return this.renderToken(details);
```

Comment:
suggestion to reduce lines and remove comment 
```suggestion
      return token ? this.renderToken(token) : this.renderToken(details);
```